Gemfile

```diff
@@ -21,7 +21,7 @@ when 'mysql'
 when 'postgresql'
   gem 'pg'
 else
-  gem 'sqlite3'
+  gem 'sqlite3', '~> 1.7'
```

Comment:
We usually have `~> 1.4`

Reply:
It's the recent 1.x version 🤷🏻 
https://rubygems.org/gems/sqlite3/versions